app/Resources/views/SonataAdmin/base_layout.html.twig

```diff
@@ -0,0 +1,16 @@
+{% extends 'SonataAdminBundle::standard_layout.html.twig' %}
+
+{% block stylesheets %}
+    {{ parent() }}
+{% endblock %}
+
+{% block side_bar_after_nav %}
+{% endblock %}
+
+{% block javascripts %}
+
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.0/jquery.min.js"></script>
```

Comment:
There is a problem... It dont work from config... I dont understand why, but when I try to do it in config its fail...